addon/mixins/in-viewport.js

```diff
@@ -91,14 +91,23 @@ export default Mixin.create({
       return;
     }
 
-    const $contextEl = $(context);
+    const contextBoundingClientRect = context.document.documentElement.getBoundingClientRect();
     const boundingClientRect = element.getBoundingClientRect();
+    let width, height;
+    if (!('width' in contextBoundingClientRect)) {
+      // IE8 and lower:
+      width = contextBoundingClientRect.right - contextBoundingClientRect.left;
+      height = contextBoundingClientRect.bottom - contextBoundingClientRect.top;
```

Comment:
@offirgolan implemented your suggestion.

note: this does mean that IE<=8 users will get the rounding bug.

there is an alternative implementation that I probably should have pushed first: just round everything to whole numbers with `Math.round` in the `isInViewport` function. the downside is that you lose subpixel accuracy with the in-viewport detection. but 99.999% of users probably don't care about it.

To make it clearer, I could remove these commits and change `isInViewport` as follows (just adding the `Math.round`):
```
export default function isInViewport(boundingClientRect = {}, height = 0, width = 0, tolerance = defaultTolerance) {
  const { top, left, bottom, right } = boundingClientRect;
  const tolerances = assign(defaultTolerance, tolerance);
  const {
    top: topTolerance,
    left: leftTolerance,
    bottom: bottomTolerance,
    right: rightTolerance
  } = tolerances;

  return (
    (Math.round(top) + topTolerance)       >= 0 &&
    (Math.round(left) + leftTolerance)     >= 0 &&
    (Math.round(bottom) - bottomTolerance) <= height &&
    (Math.round(right) - rightTolerance)   <= width
  );
}
``` 

Thoughts on this? One advantage is that it is really easy to test as this is pure function and there's an existing unit test for other cases. (This was actually my first thought).

@poteto thoughts?